specs/draggable.spec.jsx

```diff
@@ -59,6 +59,19 @@ describe('react-draggable', function () {
       assert(node.getAttribute('class') === 'foo react-draggable');
     });
 
+    it('should set the appropriate custom className when dragging or dragged', function () {
+      drag = TestUtils.renderIntoDocument(
+        <Draggable>
```

Comment:
So this should be `<Draggable classNameDragging="foo">`. Don't set it on the child.